### PR TITLE
Bump version: 2.1.2

### PR DESCRIPTION
**Pull request by JackMc**
This PR bumps the version of Toxiproxy to 2.1.2, including a security update.

### CODE REVIEW

**Comment by sirupsen** [comment]
👍  Make sure to tag it

**Comment by JackMc** [comment]
Associated homebrew PR: https://github.com/Shopify/homebrew-shopify/pull/54

**Comment by JackMc** [comment]
PR merge timeline:

RC:
- https://github.com/Shopify/ci-services/pull/50
- https://github.com/Shopify/railgun-common-services/pull/139
- https://github.com/Shopify/dev/pull/1047

Actual release:
- https://github.com/Shopify/toxiproxy/releases/tag/untagged-66b5246b57b680cc55c8 release
- https://github.com/Shopify/homebrew-shopify/pull/54

File: README.md [hunk 1]
```diff
@@ -199,8 +199,8 @@ binaries and system packages for your architecture.
 **Ubuntu**
 
 ```bash
-$ wget -O toxiproxy-2.1.1.deb https://github.com/Shopify/toxiproxy/releases/download/v2.1.1/toxiproxy_2.1.1_amd64.deb
-$ sudo dpkg -i toxiproxy-2.1.1.deb
+$ wget -O toxiproxy-2.1.2.deb https://github.com/Shopify/toxiproxy/releases/download/v2.1.2/toxiproxy_2.1.2_amd64.deb
```
Comment (jpittis) [inline README.md]: Hrmm? Why is github highlighting the `1_amd64` even though that didn't change?

**Pull request merged**